src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -30,19 +30,35 @@
     /***************************************/
     // Reference Metadata inputs
     /***************************************/
-    /**
-     * Mutect2 has the ability to use COSMIC data in conjunction with dbSNP to adjust the threshold for evidence of a variant
-     * in the normal.  If a variant is present in dbSNP, but not in COSMIC, then more evidence is required from the normal
-     * sample to prove the variant is not present in germline.
-     */
-    @Argument(fullName="cosmic", shortName = "cosmic", doc="VCF file of COSMIC sites", optional = true)
-    public FeatureInput<VariantContext> cosmicFeatureInput;
 
     /**
      * A panel of normals can be a useful (optional) input to help filter out commonly seen sequencing noise that may appear as low allele-fraction somatic variants.
      */
     @Argument(fullName="normal_panel", shortName = "PON", doc="VCF file of sites observed in normal", optional = true)
-    public FeatureInput<VariantContext> normalPanelFeatureInput;
+    public FeatureInput<VariantContext> pon;
+
+    /**
+     * A resource, such as gnomAD, containing population allele frequencies of common and rare variants.
+     */
+    @Argument(fullName="germline_resource", doc="Population vcf of germline sequencing containing allele fractions", optional = true)
+    public FeatureInput<VariantContext> germlineResource;
+
+    /**
+     * Population allele fraction assigned to alleles not found in germline resource.
+     */
+    @Argument(fullName="af_of_alleles_not_in_resurce", shortName = "default_af",
```

Comment:
I agree with this.  As long as the resource has `AC` or `AN` as well as `AF`, or a list of samples, this is easy  The difficulty is that a mixed exome/genome resource may be built out of more exome samples than genome samples, so this isn't really a single number.  I'll file an issue.